### ISSUE TITLE
No function to add custom repos like kde-unstable

### ISSUE THREAD

**Issue by philmmanjaro** [manjaro/manjaro-iso-action#23]
In the old CI script we used a simple sed cmd to add extra repos:

`sudo sed -i -e 's/\[core\]/\[kde-unstable\]\nSigLevel = PackageRequired\nInclude = \/etc\/pacman\.d\/mirrorlist\n\n\[core\]/' /usr/share/manjaro-tools/pacman-multilib.conf`

Since the [switch](https://github.com/manjaro/plasma-daily/commit/e760e3e43b1275b52a7c30031dbea47479676543#diff-2cb40e9d845122bd073c7958348d6b563cc97b44321a2994a721a945a275ba7fL139) to **manjaro-iso-action** this not given anymore. It might be better to add a function to **buildiso**, similar to **chrootbuild**, so it would be more flexible and utilize the resulting flag here.

**Comment by boredland** [comment]
Shouldn't this be done via buildiso repo.conf?

**Comment by philmmanjaro** [comment]
How you do it with Sway extra repo?

**Comment by boredland** [comment]
https://github.com/Manjaro-Sway/iso-profiles/blob/sway/community/sway/user-repos.conf

**Comment by philmmanjaro** [comment]
Have to see if we use a separate profile. Also the repo needs to be on top of the normal ones.

**Comment by boredland** [comment]
>  Also the repo needs to be on top of the normal ones.

that doesn't work then. BUT i think there is valid use cases for it. should we add that to buildiso instead of hacking it here?

**Comment by oberon-manjaro** [comment]
I agree that implementing in buildiso will be the clean solution. For now we can use this hack - can even remain on its dedicated branch here, as currently only the plasma-daily builds rely on a custom-repo ...
https://github.com/manjaro/manjaro-iso-action/commit/d5fe6da01204bbe45f960a02be864f0cd2778298
